client/apps/edit2/components/admin/components/article/index.jsx

```diff
@@ -0,0 +1,211 @@
+
+    const alreadyFetched = pluck(fetchedItems, 'id')
+    const idsToFetch = difference(related_article_ids, alreadyFetched)
+    let newItems = clone(fetchedItems)
+
+    if (idsToFetch.length) {
+      request
+        .get(`${apiURL}/graphql`)
+        .set({
+          'Accept': 'application/json',
+          'X-Access-Token': (user && user.access_token)
+        })
+        .query({ query: RelatedArticleQuery(idsToFetch) })
+        .end((err, res) => {
+          if (err) {
+            console.error(err)
+          }
+          newItems.push(res.body.data.articles)
+          const uniqItems = uniq(flatten(newItems))
+          cb(uniqItems)
+        })
+    } else {
+      return fetchedItems
+    }
+  }
+
+  render () {
+    const {
+      article,
+      apiURL,
+      channel,
+      onChangeArticleAction
+    } = this.props
+
+    return (
+      <div>
+        <ArticleAuthors />
+
+        <Row>
+          <Col xs={6}>
+            <Row>
+              <Col xs={6}>
+                <div className='field-group'>
+                  <label>Article Tier</label>
+                  <ButtonGroup>
+                    <Button
+                      active={article.tier === 1}
+                      onClick={() => onChangeArticleAction('tier', 1)}
+                    >
+                      Tier 1
+                    </Button>
+                    <Button
+                      active={article.tier === 2}
+                      onClick={() => onChangeArticleAction('tier', 2)}
+                    >
+                      Tier 2
+                    </Button>
+                  </ButtonGroup>
+                </div>
+              </Col>
+
+              <Col xs={6}>
+                <div className='field-group'>
+                  <label>Magazine Feed</label>
+                  <ButtonGroup>
+                    <Button
+                      active={article.featured}
+                      onClick={() => onChangeArticleAction('featured', true)}
+                    >
+                      Yes
+                    </Button>
+                    <Button
+                      active={!article.featured}
+                      onClick={() => onChangeArticleAction('featured', false)}
+                    >
+                      No
+                    </Button>
+                  </ButtonGroup>
+                </div>
+              </Col>
+            </Row>
+
+            {channel.type === 'editorial' &&
+              <div className='field-group'>
+                <label>Article Layout</label>
+                <ButtonGroup>
+                  <Button
+                    active={article.layout === 'standard'}
+                    onClick={() => onChangeArticleAction('layout', 'standard')}
+                  >
+                    Standard
+                  </Button>
+                  <Button
+                    active={article.layout === 'feature'}
+                    onClick={() => onChangeArticleAction('layout', 'feature')}
```

Comment:
My bad if you address this later on, but does still have the `alert()` feature somewhere?

Reply:
Ooh good point - I will add this back

Reply:
We are not actually removing any data here now that things are in redux-- do you think it is better to remove it (potentially will never display if changed to standard), or add an alert and manually delete info?